ci-operator/step-registry/baremetal/lab/agent/day2/baremetal-lab-agent-day2-ref.yaml

```diff
@@ -0,0 +1,74 @@
+ref:
+  as: baremetal-lab-agent-day2
+  from_image:
+    namespace: ci
+    name: "baremetal-qe-base"
+    tag: latest
+  commands: baremetal-lab-agent-day2-commands.sh
+  resources:
+    requests:
+      cpu: '1'
+      memory: 128Mi
+  grace_period: 600
```

Comment:
Any reason for a 10m grace period?

Reply:
Removing...

Reply:
if we use trap in the step, you'd just reduce to a lower values (e.g., 60s).

Reply:
I am not sure...It is copy from day1 install, reduce it to 60